README.md

```diff
@@ -55,15 +55,18 @@ func main() {
 		panic(err)
 	}
 
-	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
+	err = c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
+        if err != nil {
```

Comment:
mix of tabs and space here, use tabs as that is what was used previously.

Reply:
Done!

Reply:
It still has whitespaces for the new code and tabs for the old.